test/util/Util.ts

```diff
@@ -26,3 +35,91 @@ export const call = async(handler: HttpHandler, requestUrl: URL, method: string,
 
   return response;
 };
+
+export const callFile = async(
+  handler: HttpHandler,
+  requestUrl: URL,
+  method: string,
+  headers: IncomingHttpHeaders,
+  data?: Buffer,
+): Promise<MockResponse<any>> => {
+  const request = data ?
+    (streamifyArray([ data ]) as HttpRequest) :
+    streamifyArray([]) as HttpRequest;
+
+  request.url = requestUrl.pathname;
+  request.method = method;
+  request.headers = headers;
+  request.headers.host = requestUrl.host;
+  const response: MockResponse<any> = createResponse({
+    eventEmitter: EventEmitter,
+  });
+
+  const endPromise = new Promise((resolve): void => {
+    response.on('end', (): void => {
+      expect(response._isEndCalled()).toBeTruthy();
+      resolve();
+    });
+  });
+
+  await handler.handleSafe({ request, response });
+  await endPromise;
+
+  return response;
+};
+
+export const setAcl = async(
```

Comment:
I think I'm seeing double, is there a difference between this one and the one in TestHelpers?

Reply:
No, this will be removed after freyas rebase